src/store/Namespace.ts

```diff
@@ -149,6 +149,7 @@ export default {
             const linkedAddress = await getLinkedAccountPromise;
 
             commit('linkedAddress', linkedAddress);
+            return linkedAddress;
```

Comment:
I don't think you need a return here, because it already save in the `linkedAddress` state